test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -57,55 +55,47 @@ public ServerContextBuilder() {
     }
 
     public ServerContext build() {
-        ImmutableMap.Builder<String,Object> builder =
-                new ImmutableMap.Builder<String, Object>()
-                .put("--initial-token", initialToken)
-                .put("--single", single)
-                .put("--memory", memory)
-                .put("--Threads", numThreads)
-                .put("--HandshakeTimeout", handshakeTimeout)
-                .put("--sequencer-cache-size", seqCache)
-                .put("--log-size-quota-percentage", logSizeLimitPercentage)
-                .put("--batch-size", batchSize)
-                .put("--metadata-retention", retention)
-                .put("--compression-codec", compressionCodec);
+
+        ServerConfiguration conf = new ServerConfiguration()
+                .setSingleMode(single)
+                .setInMemoryMode(memory)
+                .setHandshakeTimeout(handshakeTimeout)
+                .setSequencerConflictWindowSize(seqCache)
+                .setLogSizeQuota(logSizeLimitPercentage)
```

Comment:
missing numThreads?

Reply:
"0" was using the default number and so will this.